### ISSUE TITLE
2.7.0-beta.1 tables browsing is leading to insta console crash.

### ISSUE THREAD

**Issue by efstathiosntonas** [hasura/graphql-engine#8493]
### Version Information

Server Version: 2.7.0-beta.1.cli-migrations-v3
CLI Version (for CLI related issue):

### What is the expected behaviour?
Load tables without crashing

After upgrading to 2.7.0-beta.1.cli-migrations-v3 the console table browsing breaks. It throws this error:

```

Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Bicon%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Di (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14194357)
    at https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14199310
    at Zo (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14208842)
    at Ql (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14249321)
    at Il (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14240294)
    at Rl (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14240219)
    at El (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14237069)
    at https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14188041
    at e.unstable_runWithPriority (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14263852)
    at oi (https://graphql-engine-cdn.hasura.io/console/assets/channel/beta/v2.7/vendor.js.gz:1:14187750)

```

witch translates to:

`Objects are not valid as a React child (found: object with keys {icon}). If you meant to render a collection of children, use an array instead.`

Tried to clear cache, tried it on an incognito chrome window, error persists. On my local dev environment I do not encounter such a crash.

### How to reproduce the issue?

1. Upgrade from 2.6.0 to 2.7.0-beta.1
2. Click on a table on the tables sidebar
3. Insta crash with the error mentioned above.

**Comment by JordyMiggelbrinkBridge** [comment]
We have experienced this behavior too, followed the same upgrading steps.

**Comment by binyaminadzz** [comment]
Yea, it crushes on some tables.

**Comment by efstathiosntonas** [comment]
May I ask why it takes so long for Hasura team to address this? It seems a really easy fix, it makes 2.7.1-beta.1 useless, how do you expect us to locate any other issues while we're not able to use the console? I understand that this is a beta version but it's a useless beta version.

**Comment by rikinsk** [comment]
Hi. We are working on a fix for this.

**Comment by afmotta** [comment]
This was fixed with this comment: https://github.com/hasura/graphql-engine/commit/4929d3135aadfcdff66d4f27b4d4714d57a8901a
It will be released in v2.7 stable
